planning/mission_planner/src/lanelet2_plugins/default_planner.cpp

```diff
@@ -145,8 +145,8 @@ void DefaultPlanner::initialize_common(rclcpp::Node * node)
     node_->create_publisher<MarkerArray>("debug/goal_footprint", durable_qos);
 
   vehicle_info_ = vehicle_info_util::VehicleInfoUtil(*node_).getVehicleInfo();
-  param_.goal_angle_threshold_deg = node_->declare_parameter("goal_angle_threshold_deg", 45.0);
-  param_.enable_correct_goal_pose = node_->declare_parameter("enable_correct_goal_pose", false);
+  param_.goal_angle_threshold_deg = node_->declare_parameter("goal_angle_threshold_deg");
+  param_.enable_correct_goal_pose = node_->declare_parameter("enable_correct_goal_pose");
```

Comment:
A template type is necessary.

Reply:
@isamu-takagi Thanks for your comment.

@yamazakiTasuku Could you fix it?

Reply:
I fixed in https://github.com/autowarefoundation/autoware.universe/pull/3127/commits/2e7e30ae007d36da3b199c15b88a92673058e8a6.